mmv1/templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -1,19 +1,50 @@
 func rrdatasDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
-	if d.Get("type") == "AAAA" {
-		return ipv6AddressforDnsDiffSuppress(k, old, new, d)
+	oo, nn := d.GetChange("rrdatas")
+	if oo == nil || nn == nil {
+		return false
 	}
-	if d.Get("type") == "MX" {
-		return (strings.ToLower(old) == strings.ToLower(new))
-	}
-	if d.Get("type") == "TXT" {
-		return strings.ToLower(strings.Trim(old, `"`)) == strings.ToLower(strings.Trim(new, `"`))
+
+	olist := convertStringArr(oo.([]interface{}))
+	nlist := convertStringArr(nn.([]interface{}))
+
+	parsefunc := func(x string) string {
```

Comment:
`parsefunc` should be `parseFunc`, and lets use a more descriptive name than `x` here, maybe `record`?